network/server/tchannelthrift/thrift/rpc.thrift

```diff
@@ -50,7 +50,7 @@ service Node {
 	void write(1: WriteRequest req) throws (1: WriteError err)
 	void writeBatch(1: WriteBatchRequest req) throws (1: WriteBatchErrors err)
 	FetchResult fetch(1: FetchRequest req) throws (1: Error err)
-	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req) throws (1: Error err)
+	FetchRawBatchResult fetchRawBatch(1: FetchRawBatchRequest req)
```

Comment:
Right yea, but we have cases where `FetchRawBatch` just returns a `nil` and an error, in which case we don't have a batch result object to store the error?

Also, I noticed the `FetchRawBatch` method in the `Node` interface in tchan-rpc.go still return an error even if `throw (1: Error err)` is removed here. Does the generated method always return an error no matter whether we explicitly specify it or not?